src/sagemaker/workflow/steps.py

```diff
@@ -548,10 +552,12 @@ def properties(self):
 
     def to_request(self) -> RequestType:
         """Updates the request dictionary with cache configuration."""
+        """transfer the is_final_step  parameter to json."""
```

Comment:
This may break the doc string generation, i.e. `tox -e doc8` or `tox -e sphnix` 

Let's remove line 555 and update the line 554 and also make it short, e.g. "Update the request dictionary with cache configuration etc."